### PR TITLE
Automated Black Updates

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
- Creates an action to automate the black update
- Run black for full repo and the make a PR.

## Motivation
The issue caused due to the black update.

## Testing Status
Tested in my fork https://github.com/naveen521kk/manim/pull/13

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: .github/workflows/black.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        run: echo ::set-output name=modified::$(if git diff-index --quiet HEAD --; then echo "false"; else echo "true"; fi)
+
+      - name: Print Black Version if Failed
+        shell: bash
+        if: steps.git-check.outputs.modified == 'true'
+        run: |
+          blackVer=$(black --version)
+          blackVersion=$(python -c "print('${blackVer}'.split()[2])")
+          date=$(python -c "import time;print(time.localtime().tm_mday)")
+          pushBranch="black-${blackVersion}-${date}"
```
Comment (leotrs) [inline .github/workflows/black.yml:39]: Excellent.

File: .github/workflows/black.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      - name: Make a PR
+        env:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          $env:GITHUB_TOKEN = $env:access_token
+          $blackVer = black --version
+          $blackVersion = python -c "print('$($blackVer)'.split()[2])"
+          $date = python -c "import time;print(time.localtime().tm_mday)"
+          $pushBranch="black-$($blackVersion)-$($date)"
+          $postContents = @{title="Black on Master - Automation";head="ManimCommunity:$($pushBranch)";base="master";body="This an automated pull requests."} | ConvertTo-Json -Compress
```
Comment (naveen521kk) [inline .github/workflows/black.yml:65]: Maybe suggest me some good `title` and `body` :).

File: .github/workflows/black.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        run: |
+          blackVer=$(black --version)
+          blackVersion=$(python -c "print('${blackVer}'.split()[2])")
+          date=$(python -c "import time;print(time.localtime().tm_mday)")
+          pushBranch="black-${blackVersion}-${date}"
+          git config user.name "github-actions"
+          git config user.email "actions@github.com"
+          git checkout -B "$pushBranch"
+          git add .
+          git commit -m "Running Black ON Master"
```
Comment (huguesdevimeux) [inline .github/workflows/black.yml:44]: Nitpick (like, really a nitpick)
```suggestion
          git commit -m "Running Black on Master"
```

File: .github/workflows/black.yml (hunk 2 above)
Comment (huguesdevimeux) [reply .github/workflows/black.yml:65]: Maybe "updating Black to the latest version available [{version})" ? 

And for the body, I don't know, a more descriptive thing, like what is this PR, why etc.

**Review by huguesdevimeux — APPROVED** [review]
Brilliant

File: .github/workflows/black.yml (hunk 3 above)
Comment (naveen521kk) [reply .github/workflows/black.yml:44]: I did for fun and was intentional. lol

Comment (naveen521kk) [reply .github/workflows/black.yml:44]: I will change it soon.

File: .github/workflows/black.yml (hunk 2 above)
Comment (leotrs) [reply .github/workflows/black.yml:65]: Wait I'm confused. This is for the PR, no? The PR doesn't update the black version, it only runs black and commits the differences, correct?

If so, the title could be "Automated PR: autoformat with black". The body may contain the list of files changed, if that's at all possible. If not, just leave the body empty.

Comment (naveen521kk) [reply .github/workflows/black.yml:65]: Please wait for me to do some changes here. I will make if more readable.

**Comment by naveen521kk** [comment]
Please merge this only when I updraft it.

**Comment by naveen521kk** [comment]
It now ready and show file name 
cc https://github.com/naveen521kk/manim/pull/16

**Pull request merged**